docs/manual/developer/03_creating_content.md

```diff
@@ -411,8 +411,8 @@ MAKEFILE_ID_TO_PRODUCT_MAP = {
 
 
 5. Create a new file in the product directory called `CMakeLists.txt`:
-```
-cat << EOF >> $NEW_PRODUCT/CMakeLists.txt
```

Comment:
I see multiple instances of such changes in this PR. But I think they are not correct - this command is using Bash heredoc to insert contents into a file and the change you propose breaks it.
Heredoc is denoted with <<.
https://www.howtogeek.com/719058/how-to-use-here-documents-in-bash-on-linux/